ClientRuntimes/CSharp/ClientRuntime.Etw/ClientRuntime.Etw.csproj

```diff
@@ -30,14 +30,14 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\ClientRuntime\ClientRuntime.csproj">
-      <Project>{7f524559-93f9-4f3f-9e2c-af41a0c2e6f4}</Project>
+      <Project>{7F524559-93F9-4F3F-9E2C-AF41A0C2E6F4}</Project>
       <Name>ClientRuntime</Name>
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <CodeAnalysisDictionary Include="$(SolutionDir)tools\CustomDictionary.xml">
+    <CodeAnalysisDictionary Include="%24%28SolutionDir%29Tools\CustomDictionary.xml">
```

Comment:
do we have to use the escape character? I feel we might have some other issues going on